src/components/views/messages/TextualBody.tsx

```diff
@@ -388,6 +388,14 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
             return false;
         }
 
+        const url = node.getAttribute("href");
+        const host = url?.match(/^https?:\/\/(.*?)(\/|$)/)?.[1];
```

Comment:
Why a regex rather than `new URL(url).host`?

Reply:
This is the existing implementation, didn't fancy poking the bear. I just swapped the order of 2 blocks of code

Reply:
oh, fair enough